### ISSUE TITLE
Can't pull docker image anymore

### ISSUE THREAD

**Issue by digitalfredy** [terraform-linters/tflint#1718]
### Summary

Context
=======

I'm using the docker image in my CI and it was working 30m ago (not sure if my CI pull at any time or if it save the images) and is not working anymore.

Expected
=======

I should be capable to pull the image as before

Extra information
==============

Thanks for doing this OpenSource project.

### Command

docker pull ghcr.io/terraform-linters/tflint

### Terraform Configuration

```terraform
N/A
```

### TFLint Configuration

```HCL
N/A
```

### Output

```Shell
Using default tag: latest
Error response from daemon: Head "https://ghcr.io/v2/terraform-linters/tflint/manifests/latest": Get "https://ghcr.io/token?scope=repository%3Aterraform-linters%2Ftflint%3Apull&service=ghcr.io": net/http: request canceled (Client.Timeout exceeded while awaiting headers)
```

### TFLint Version

latest

### Operating System

- [X] Linux
- [ ] macOS
- [ ] Windows
